README.md

```diff
@@ -25,7 +25,8 @@ from the lovely people at deadsnakes. Then it will install the needed packages t
 
 ### One-Time Installation
 
-From this repository, run `sudo make install`. This will install the `etc-as` as downloaded to your machine.
+From this repository, run `make build && sudo make install`.
+This will install the `etc-as` as downloaded to your machine.
```

Comment:
Why does it feel weird? Maybe it's because I wrote it but it sounds fine.

Reply:
I would expect something like `the etc-as command/script`. Also, "as downloaded" is a conditional postfix that I had to read three times that it meant it will not download more stuff. But if you think it's fine, leave it be.

Reply:
Yeah, this was also an issue before, but apparently noone noticed.